src/dbnode/storage/shard_race_prop_test.go

```diff
@@ -33,12 +33,12 @@ import (
 	"github.com/m3db/m3/src/x/context"
 	"github.com/m3db/m3/src/x/ident"
 	xtime "github.com/m3db/m3/src/x/time"
+	"github.com/m3db/m3/src/dbnode/storage/series"
```

Comment:
err `importOrder` not picking this up either?

Reply:
yep it isn't.

Reply:
lets address metalint running separate from this PR

Reply:
Yeah definitely.